<launch>
	<!--启动运动补偿节点 -->
	<node  pkg="datasync" type="Initial_Motion_Compensation" name="Initial_Motion_Compensation" output="screen">
		
		<!-- dvxplorer config-->
		<param name="weight_param"    value="640"/>
		<param name="height_param"    	value="480"/> 
		<param name="focus"           	value="3900"/>
		<param name="pixel_size"      	value="9"/> 


		<!-- davis 346 config
		<param name="weight_param"          value="346"/>
		<param name="height_param"          value="260"/>
		<param name="focus"           		value="6550"/>
		<param name="pixel_size"      		value="18.5"/> -->


		<!-- davis 240c config-->
		<!-- <param name="weight_param"    value="240"/>
		<param name="height_param"    	value="180"/> 
		<param name="focus"          	value="4000"/>
		<param name="pixel_size"      	value="18.5"/> -->


		<!-- davis 240b config-->
		<!-- <param name="weight_param"    value="190"/>
		<param name="height_param"    	value="180"/> 
		<param name="focus"           	value="4000"/>
		<param name="pixel_size"      	value="18.5"/> -->


	</node>
	<!-- Start rviz visualization with preset config -->
  	<!-- <node pkg="rviz" type="rviz" name="rviz" args="-d $(find datasync)/launch/dvxplorer.rviz"/> -->
</launch>


	
